File: marios/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int height;

    do
    {
        height = get_int("Height: ");

    }
    while (height < 0 || height > 23);

    for(int i = 0; i < height; i++)
    {

        // Print the empty spaces to push pyramid ot right
        for (int k = 1; k <= height - 1 - i; k++)
        {
            printf(" ");
        }

        // Print hashtags
        for (int j = 0; j < i + 1 + 1; j++)
        {

            printf("#");
        }

        // Start a new line
        printf("\n");
    }

}

File: pset6/marios/more/mario.py
from cs50 import get_int

def main():

    while True:
        height = get_int("Height between 0 and 23: ")

        if (height >= 0 or height <= 23):
            break;

    halfPyramid(height)


def halfPyramid(height):
    for i in range(height):
        for j in range(1, height - 1 - i):
            print(" ", end = "")
        for k in range(1, i + 2):
            print("#", end = "")

        for k in range(1, i + 2):
            print("#", end = "")
        print(" ")





if __name__ == "__main__":
    main()